### ISSUE TITLE
Dashboard cells bleed over one another...

### ISSUE THREAD

**Issue by MrJoy** [contribsys/faktory#249]
https://cl.ly/59475b2892e0

Perhaps there's something that could be done like scaling the font size with the window width, or making content wrap instead of spill?

**Comment by mperham** [comment]
Yeah, it happens at that awkward point where the phone UI hasn't kicked in yet but the window downsizes to a medium size which doesn't work well with larger numbers. CSS really isn't my strong suit so this is unlikely to be fixed unless someone else wants to investigate.

**Comment by scottrobertson** [comment]
We will fix this when we eventually get to that number :)
